# Sun 2005-01-15 - Jason Watson
# while trying to build on RedHat Enterprise 3 with automake-1.6, I ran into this error from automake.
# the following patch resolved the issue by simply renaming the var in configure.ac
# uname -a = Linux localhost.localdomain 2.4.21-15.EL #1 Thu Apr 22 00:27:41 EDT 2004 i686 i686 i386 GNU/Linux
# automake --verson = automake (GNU automake) 1.6.3
###################################################################
#
# $ automake --add-missing
# configure.ac:23: `automake requires `AM_CONFIG_HEADER', not `AC_CONFIG_HEADER'
#
###################################################################
diff -urN libhid-0.2.10.orig/configure.ac libhid-0.2.10.new/configure.ac
--- libhid-0.2.10.orig/configure.ac    2004-11-09 11:28:00.000000000 -0500
+++ libhid-0.2.10.new/configure.ac    2005-01-12 23:49:14.000000000 -0500
@@ -20,7 +20,7 @@
AC_INIT(MD_INIT_NAME, MD_INIT_VERSION, MD_INIT_ADDRESS)
-AC_CONFIG_HEADER([config.h])
+AM_CONFIG_HEADER([config.h])
AC_CONFIG_SRCDIR([include/hid.h])
AC_CONFIG_AUX_DIR([.])
